modules/test/playwright/tests/commerce/commerce-currency-web/multiCurrency.spec.ts

```diff
@@ -0,0 +1,689 @@
+	{tag: ['@LPD-48196']},
+	async ({
+		apiHelpers,
+		commerceAdminChannelDetailsCurrenciesPage,
+		commerceAdminChannelDetailsPage,
+		commerceAdminChannelsPage,
+		commerceMiniCartPage,
+		commerceOrdersPage,
+		commerceThemeMiniumCatalogPage,
+		page,
+		productDetailsPage,
+	}) => {
+		test.setTimeout(180000);
+		const {catalog, channel, site} = await classicCommerceSetUp(
+			apiHelpers,
+			getRandomString()
+		);
+
+		const account = await apiHelpers.headlessAdminUser.postAccount({
+			name: getRandomString(),
+			type: 'business',
+		});
+		apiHelpers.data.push({id: account.id, type: 'account'});
+
+		const user =
+			await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+				'demo.unprivileged@liferay.com'
+			);
+		const rolesResponse =
+			await apiHelpers.headlessAdminUser.getAccountRoles(account.id);
+
+		const accountRoleBuyer = rolesResponse?.items?.filter((role) => {
+			return role.name === 'Buyer';
+		});
+
+		const currencies =
+			await apiHelpers.headlessCommerceAdminCatalog.getCurrenciesPage('');
+
+		const currency1 = currencies.items.find((item) =>
+			item.name['en_US'].includes('US Dollar')
+		);
+		const currency2 = currencies.items.find((item) =>
+			item.name['en_US'].includes('Australian Dollar')
+		);
+		const currency3 = currencies.items.find((item) =>
+			item.name['en_US'].includes('Euro')
+		);
+
+		const currencyName1 = currency1.name['en_US'];
+		const currencyName2 = currency2.name['en_US'];
+		const currencyName3 = currency3.name['en_US'];
+
+		const product = (
+			await apiHelpers.headlessCommerceAdminCatalog.getProducts(
+				new URLSearchParams({
+					filter: `name eq 'U-Joint'`,
+					nestedFields: `skus`,
+				})
+			)
+		).items[0];
+
+		const productSku = (
+			await apiHelpers.headlessCommerceDeliveryCatalog.getChannelProductSkusPage(
+				channel.id,
+				product.productId,
+				new URLSearchParams({
+					nestedFields: `price`,
+				})
+			)
+		).items[0];
+
+		const cart = await apiHelpers.headlessCommerceDeliveryCart.postCart(
+			{
+				accountId: account.id,
+				cartItems: [
+					{
+						quantity: 1,
+						skuId: product.skus[0].id,
+					},
+				],
+			},
+			channel.id
+		);
+
+		const priceList =
+			await apiHelpers.headlessCommerceAdminPricing.postBasePriceList({
+				catalogId: catalog.id,
+				currencyCode: currency3.code,
+				name: getRandomString(),
+				type: 'price-list',
+			});
+
+		const priceEntry =
+			await apiHelpers.headlessCommerceAdminPricing.postPriceEntry({
+				price: 100,
+				priceListId: priceList.id,
+				skuId: product.skus[0].id,
+			});
+
+		await apiHelpers.headlessAdminUser.assignAccountRoles(
+			account.externalReferenceCode,
+			accountRoleBuyer[0].id,
+			user.emailAddress
+		);
+		const siteRole =
+			await apiHelpers.headlessAdminUser.getRoleByName('Site Member');
+		await apiHelpers.headlessAdminUser.assignUserToSite(
+			siteRole.id,
+			site.id,
+			user.id
+		);
+		await apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
```

Comment:
new line 

also, why in this case these creation are "after the products"? compared with the previous test where are before